code/tINIT/prepHumanModelForftINIT.m

```diff
@@ -0,0 +1,126 @@
+function prepDataHumanGEM = prepHumanModelForftINIT(model, convertGenes)
+% prepHumanModelForftINIT
+%   Generates a "standard" prepData structure from Human-GEM to be used in ftINIT.
+%   This function typically takes a couple of hours to run, so we recommend to only 
+%   run this once and save the results to a .mat file.
+%
+%   convertGenes            True if the genes should be converted to symbols 
+%                           ('CD8A' etc., and not ENSEMBL)
+%
+%   Usage: prepDataHumanGEM = prepHumanModelForftINIT(convertGenes)
+%
+
+
+
+
+%load tasks
+taskStruct = parseTaskList(which('metabolicTasks_Essential.txt'));
+%Spontaneous reactions:
+rxns_tsv = importTsvFile(which('reactions.tsv'));
+spont = rxns_tsv.spontaneous;
+spontRxnNames = rxns_tsv.rxns(spont == 1);%very few
+
+%remove some reactions often not used from the model to speed up calculations
+
+%drug reactions:
+m = removeDrugReactions(model);
+%remove all AA triplet rxns, i.e. rxns of the type
+%2 H2O[c] + Tryptophanyl-Glycyl-Aspartate[c] <=> aspartate[c] + glycine[c] + tryptophan[c]
+%This is only used for import of such compounds, and if you don't have that in your "medium",
+%which we usually don't, these are pretty pointless.
+AATriplets = getAATripletReactions(m,false);
+%length(AATriplets)%735
+m = removeReactions(m, AATriplets);
+
+%Remove the duplicate complex I reaction with ROS - it only causes problems to include it
+m = removeReactions(m, {'MAR13081'});
+
+
+%prepare some reactions that can always be on:
+
+%protein reactions creation/degradation
+
+proteinRxns = { ...
+    'MAR05155'; ...
+    'MAR05156'; ...
+    'MAR05161'; ...
+    'MAR05167'; ...
+    'MAR05168'; ...
+    'MAR05169'; ...
+    'MAR05170'; ...
+    'MAR05171'; ...
+    'MAR05172'; ...
+    'MAR05174'; ...
+    'MAR05260'; ...
+    'MAR05262'; ...
+    'MAR05264'; ...
+    'MAR05266'; ...
+    'MAR05267'; ...
+    'MAR05268'; ...
+    'MAR05269'; ...
+    'MAR05270'; ...
+    'MAR05271'; ...
+    'MAR05273'; ...
+    'MAR05275'; ...
+    'MAR05277'; ...
+    'MAR05279'; ...
+    'MAR05281'; ...
+    'MAR05283'; ...
+    'MAR05291'; ...
+    'MAR09817'; ...
+    'MAR09818'};
+
+%reactions that just pool metabolites
+poolRxns = { ...
+    'MAR00011'; ...
+    'MAR00012'; ...
+    'MAR00477'; ...
+    'MAR05233'; ...
+    'MAR05234'; ...
+    'MAR05238'; ...
+    'MAR05239'; ...
+    'MAR05243'; ...
+    'MAR05244'; ...
+    'MAR05247'; ...
+    'MAR09022'; ...
+    'MAR00015'; ...
+    'MAR00016'; ...
+    'MAR00017'; ...
+    'MAR10033'; ...
+    'MAR10035'; ...
+    'MAR10036'; ...
+    'MAR10037'; ...
+    'MAR10038'; ...
+    'MAR10062'; ...
+    'MAR10063'; ...
+    'MAR10064'; ...
+    'MAR10065'; ...
+    'MAR13082'};
```

Comment:
There is a lot of hard-coded items in this function, which I generally would like to avoid. But since I don't really have better suggestions, I think it's fine to leave like this.